Package.json listed as dependency

https://github.com/bfwg/ngx-drag-scroll/blob/develop/package.json#L81

I believe this is causing this error when publishing as a dependency
npm ERR! argv "/usr/local/Cellar/node/5.9.1/bin/node" "/usr/local/bin/npm" "publish"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! code EISDIR
npm ERR! errno -21
npm ERR! syscall read

npm ERR! eisdir EISDIR: illegal operation on a directory, read
npm ERR! eisdir This is most likely not a problem with npm itself
npm ERR! eisdir and is related to npm not being able to find a package.json in
npm ERR! eisdir a package you are trying to install.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+### 1.5.4
+
+Issue #89 - Remove unused dependency.
+
 ### 1.5.3
 
 Change compile target to es5 which will make the library work on IE 11.

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ngx-drag-scroll",
-  "version": "1.5.3",
+  "version": "1.5.4",
   "description": "Lightweight drag to scroll directive for Angular2+",
   "contributors": [
     {
@@ -76,8 +76,5 @@
     "tslint": "5.8.0",
     "typescript": "2.4.2",
     "zone.js": "0.8.18"
-  },
-  "dependencies": {
-    "package.json": "^2.0.1"
   }
 }

